src/app/_utils/getCountForResetCategory.ts

```diff
@@ -0,0 +1,13 @@
+import { DEFAULT_CATEGORY } from '@/components/Category'
+
+import { CategoryCounts } from '@/types/categoryTypes'
+
+export function getCountForResetCategory(counts: CategoryCounts) {
+  const totalSum = Object.values(counts).reduce((sum, value) => sum + value, 0)
+
+  if (totalSum !== 0) {
+    return {
+      [DEFAULT_CATEGORY]: totalSum,
+    }
+  }
+}
```

Comment:
I don't understand this function. What is its purpose?

Reply:
The purpose is to get a count for `DEFAULT_CATEGORY`, which in our case is "All", so we pass `categoryCounts` as a prop , for example `{ "use_case": 12, "interview": 33 }` and as a result we get an object, for example `{"All": 79 }`